boards/arm/cy8ckit_062_ble/cy8ckit_062_ble_m4.dts

```diff
@@ -28,4 +28,6 @@
 &uart6 {
 	status = "okay";
 	current-speed = <115200>;
+
+	pinctrl-0 = <&p13_0_uart6_rx &p13_1_uart6_tx>;
```

Comment:
The commit message talks about uart-5 but here we're updating uart-6. Just checking if it's intentional.

Reply:
No it is not. This code was written with Zephyr 2.3 and was reworked many times. I think missed commit messages after squash. Glad you saw that, thanks!